components/molecules/UniversalBenefitCard.js

```diff
@@ -24,20 +42,34 @@ export default function UniversalBenefitCard(props) {
               />
             )
           })}
-      {props.benefit.taskGroups == null || props.benefit.taskGroups.length <= 0
-        ? null
-        : props.benefit.taskGroups.map((taskList, index) => {
-            return (
-              <div key={index}>
-                <BenefitTasks
-                  isExpanded={true}
-                  header={taskList.Header}
-                  tasks={taskList.Tasks}
-                />
-                <HorizontalRule width="w-auto sm:w-full" />
-              </div>
-            )
-          })}
+      {props.benefit.taskGroups == null ||
+      props.benefit.taskGroups.length <= 0 ? null : (
+        <div ref={topOfTaskRef}>
+          {!isOpen
+            ? null
+            : props.benefit.taskGroups.map((taskList, index) => {
+                return (
+                  <div key={index}>
+                    <BenefitTasks
+                      isExpanded={true}
+                      header={taskList.Header}
+                      tasks={taskList.Tasks}
+                    />
+                    <HorizontalRule width="w-auto sm:w-full" />
+                  </div>
+                )
+              })}
+          <ViewMoreLessButton
+            id={props.benefit.programCode + '-card-button'}
+            onClick={() => {
+              setIsOpen(!isOpen)
+              scrollTo()
```

Comment:
instead of a scrollTo function try looking into calling scrollIntoView directly
parent.scrollIntoView()